mne/epochs.py

```diff
@@ -272,9 +272,10 @@ def __init__(self, info, data, events, event_id, tmin, tmax,
             self.preload = False
             self._data = None
         else:
+            index_offset = 1 if data.shape[-1] > 1 else 0
             assert decim == 1
             if data.ndim != 3 or data.shape[2] != \
-                    round((tmax - tmin) * self.info['sfreq']) + 1:
+                    round((tmax - tmin) * self.info['sfreq']) + index_offset:
```

Comment:
In other words, I'm worried when the single-value case needs to be exceptional. It really seems like it shouldn't need to be. The fact that we are treating it as such might indicate a bigger problem with how we deal with times, that we should fix at the root instead of implementing workarounds. Maybe that's incorrect, though.